.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Whats this file for? I keep my code settings in the base of the workspace so we don' track files like this.

Reply:
Removed